src/test/scala/org/broadinstitute/dsde/firecloud/dataaccess/MockAgoraDAO.scala

```diff
@@ -33,4 +35,46 @@ class MockAgoraDAO extends AgoraDAO {
     Future(SubsystemStatus(ok = true, None))
   }
 
+  override def batchCreatePermissions(inputs: List[EntityAccessControlAgora])(implicit userInfo: UserInfo): Future[List[EntityAccessControlAgora]] =
+    Future.successful(MockAgoraACLData.multiUpsertResponse)
+
+  override def getPermission(url: String)(implicit userInfo: UserInfo): Future[List[AgoraPermission]] = {
+    val pathString = Uri(url).path.toString()
+
+    val rawData = if (pathString.endsWith(MockAgoraACLData.standardPermsPath)) {
+      MockAgoraACLData.standardAgora
+    } else if (pathString.endsWith(MockAgoraACLData.withEdgeCasesPath)) {
+      MockAgoraACLData.edgesAgora
+    } else {
+      List.empty
+    }
+
+    // methods endpoints return the mock data in reverse order - this way we can differentiate methods vs. configs
+    if (pathString.startsWith("/api/v1/methods/")) {
+      Future.successful(rawData.reverse)
+    } else {
+      Future.successful(rawData)
+    }
```

Comment:
this is really confusing, but it migrates pre-existing test code (which I originally wrote so I claim fault)